class-banner.php

```diff
@@ -506,6 +521,11 @@ public function load_args_from_layout_content( array $raw_content, int $counter
 				$this->image = null;
 			}
 
+			// Tagline.
+			if ( true === apply_filters( 'hogan/module/banner/tagline/enabled', false ) ) {
+				$this->tagline = $raw_content['tagline'];
```

Comment:
Should define the $tagline var in top of this class.

Reply:
Label is a property that is used on the module instance. Can we keep this as tagline, that is what is used already for DSS?